tools/processresults.py

```diff
@@ -8,11 +8,13 @@
 import dateutil.parser
 from datetime import datetime
 import json
+import logging
 import re
 
 DOGFOOD_START_DATE = datetime(2014, 12, 01, 0, 0, 0, 0)
 DEMOGRAPHIC_STABLE_DATE = datetime(2014, 12, 18, 0, 0, 0, 0)
 FRIENDS_AND_FAMILY_BETA_DATE = datetime(2015, 4, 9, 0, 0, 0, 0)
+PUBLIC_LAUNCH_DATE = datetime(2015, 5, 11, 13, 30, 0, 0)
```

Comment:
You might want to add timezone information to the launch date as well.

(I don't think it's important, but since the date is very specific it might be a good idea)